src/styles/main.css

```diff
@@ -67,27 +67,89 @@ button:hover {
     background-color: var(--button-hover-color)!important;
 }
 
-.logo {
-    margin: 20px;
+.flex-container {
+    display: flex;
+    align-items: stretch;
+    flex-grow: 1;
+    overflow: auto;
+}
+  
+.flex-column {
+    margin: 2px;
+    font-size: 30px;
+    width: 150px;
+    min-height: 500px;
+}
+
+.content-column {
+    background-color: #f1f1f1;
+    text-align: center;
+    width: 150px;
+    display: flex;
+    min-height: 500px;
+    flex-direction: column;
 }
 
-.secondary-color {
-    color: var(--secondary-color);
+.header{
+    height: 35px;
 }
 
-.decoration {
-    animation: rotate 2000ms linear infinite;
+.name-column{
+    color: black;
+    font-size: 12px;
+    padding: 10px;
+    text-align: center;
 }
 
-.button, .button-cta {
-    transition: transform 500ms cubic-bezier(1, 0.01, 0.57, 1.35);
+.edit-column{
+    float: left;
+    font-size: 10px;
+    display: none;
 }
 
-.active {
-    transform: translateY(50%);
+.add-column{
+    font-size: 10px;
+    float: right;
+    display: none;
 }
 
-@keyframes rotate {
-    from {transform: rotate(0);}
-    to {transform: rotate(180deg);}
+.to-do{
+    order: 0;
 }
+
+.done{
+    order: 1111;
```

Comment:
isn't the number a bit random ?

Reply:
Daa, am dat o valoare mare aleatoare pentru a 'retine' cumva ca 'DONE' e ultima coloana

Reply:
Atunci muta valoarea intr-o variabila css :).